src/query/sql/src/planner/semantic/type_check.rs

```diff
@@ -3513,6 +3517,55 @@ impl<'a> TypeChecker<'a> {
             }
         }
 
+        let const_udf_type = if ["wasm"].contains(&udf_definition.language.to_lowercase().as_str())
```

Comment:
it's just simple comparision.

```
udf_definition.language.eq_ignore_ascii_case("wasm")

```

Reply:
Apologies for the previous code complexity. I have refactored the code to simplify it, and it should be better now.